tests/components/nuheat/test_climate.py

```diff
@@ -155,7 +153,7 @@ def test_target_temperature(self):
 
     def test_operation_list(self):
         """Test the operation list."""
-        assert self.thermostat.hvac_modes == [HVAC_MODE_HEAT, HVAC_MODE_OFF]
+        assert self.thermostat.hvac_modes == [HVAC_MODE_AUTO, HVAC_MODE_HEAT]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5218980)